partiql-ast/src/main/java/org/partiql/ast/sql/SqlDialect.kt

```diff
@@ -128,8 +128,15 @@ public abstract class SqlDialect : AstVisitor<SqlBlock, SqlBlock>() {
      * @param node
      * @param tail
      */
-    public open fun visitExprWrapped(node: Expr, tail: SqlBlock): SqlBlock = when (node) {
-        is ExprQuerySet -> {
+    public open fun visitExprWrapped(node: Expr, tail: SqlBlock, operator: Boolean = false): SqlBlock = when {
```

Comment:
self-review: not quite a fix but more of adjustment to follow how the pretty-printer for Scribe wraps expressions in parens. shouldn't be a breaking change since we provide a default value for `operator`